staging/vhost-device-sound/Cargo.toml

```diff
@@ -10,6 +10,7 @@ license = "Apache-2.0 OR BSD-3-Clause"
 edition = "2018"
 
 [features]
+xen = ["vm-memory/xen", "vhost/xen", "vhost-user-backend/xen"]
```

Comment:
I'd put this change in a separate commit

Reply:
done